hail/src/test/scala/is/hail/SparkSuite.scala

```diff
@@ -19,7 +19,11 @@ object SparkSuite {
 }
 
 class SparkSuite extends TestNGSuite {
-  lazy val hc: HailContext = SparkSuite.hc
+  lazy val hc: HailContext = {
+    if (System.getenv("HAIL_ENABLE_CPP_CODEGEN") != null)
+      SparkSuite.hc.flags.set("cpp", "1")
```

Comment:
It seems weird to me to set a flag to a string value. This seems like it could be confusing that the code above just checks whether it's null or not without checking the value. What do you think?

Reply:
yeah, I'm not super sure how to handle this. I like having the flag be able to be more than just a boolean switch, so that maybe we can start playing around with parameters somewhere, but you're right that it's odd to have this particular flag be a string if all that's being checked is the non-null-ness.